### PR TITLE
Update Readme with env variable set up

### PR DESCRIPTION
**Pull request by MartinSWDev**
README.md previous text was removed, updated with env set up

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,18 @@
-# Northcoders House of Games API
+# NC Backend Project
 
-## Background
+## Project Trello
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+https://trello.com/b/zlUxNoND/be-nc-games
```
Comment (Poonam-raj) [inline README.md:5]: You don't need to share the trello to the user - this is for you to manage the project but not useful for anyone else

File: README.md [hunk 2]
```diff
@@ -1,57 +1,18 @@
...
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+## Set Up Instructions
 
-## Kanban
+1. Environment Variables.
```
Comment (Poonam-raj) [inline README.md:9]: The user will need to know where to make these files, what to put inside them, specifically which file needs to reference which specific database. We can't assume they'll know any of this

File: README.md [hunk 3]
```diff
@@ -1,57 +1,18 @@
-# Northcoders House of Games API
+# NC Backend Project
 
-## Background
+## Project Trello
```
Comment (Poonam-raj) [inline README.md:4]: Make sure you tell the user they will also need to run 'npm install' to get all the needed dependencies

**Review by Poonam-raj — CHANGES_REQUESTED** [review]
Few amendments needed on the readme but package json looks perfect

**Review by leekli — APPROVED** [review]
Good to merge.

**Pull request merged**